docs/guide/env-vars.md

```diff
@@ -1,18 +1,18 @@
 # Environment variables
 
-This is the list of the actually supported env vars:
+This is the list of the supported environment variables:
 
-- `NETWORK_KEY`: Zwave Network key
+- `NETWORK_KEY`: Z-Wave Network key
```

Comment:
```suggestion
- `NETWORK_KEY`: Z-Wave S0 Network key
```

Reply:
What about the S2 keys?

Reply:
S2 keys doesn't have a dedicated env var yet